.luacheckrc

```diff
@@ -37,14 +37,11 @@ allow_defined_top = true
 ignore = {
     "131", -- unused global variable, we should annotate the ones we expect
     "542", -- empty if branch  (usually has a comment)
+    "212", -- unused argument; name is used as documentation
```

Comment:
hmm i'm not opposed to having this enforced. feels a bit strange to have unused named arguments to me

Reply:
There are a bunch of places like [this](https://github.com/spirit-island/spirit-island-mod/blob/beta/global-script.lua#L283-85) where there are a bunch of parameter required as part of an API, and I think it is better to name the arguments, then to need to use `_` for all the unused arguments.
```
    addHotkey("Add Fear", function (droppingPlayerColor, hoveredObject, cursorLocation, key_down_up)
        aidBoard.call("addFear")
    end)
```
If it wasn't for that, I'd agree with you.